CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
-
+- Added in WindowAttributes `center_window: bool` `start_monitor: i16`, Window centering is by default enabled on windows.
+- Added convenience functions to manage window centering, `center()` ,`set_center_monitor()`, `set_centered()` see more in window struct.
```

Comment:
Change to this instead:
"Added the following convenience functions for managing window centering: `center()` ,`set_center_monitor()`, and `set_centered()`. Refer to the documentation for how to use them."